chore: address alloydb.md friction points (#55)
GoogleCloudPlatform/database-query-extension: docs/datastore/alloydb.md

                           vpcaccess.googleapis.com \
                           aiplatform.googleapis.com
    ```

> 1. [Install python][install-python] and set up python [virtual environment][venv].

Reviewer: ```suggestion
1. [Install python][install-python] and set up a python [virtual environment][venv].
```

> 1. Make sure you have python version 3.10 and up.

Reviewer: ```suggestion
1. Make sure you have python version 3.10+ installed:
```

    ```bash
    python -V
    ```

1. Download and install [postgres-client cli (`psql`)][install-psql].

> 1. Download and install [jq][install-jq].

Reviewer: See comment on bug to use a different gcloud command instead of adding this additional tool

## Create a AlloyDB cluster

1. Set environment variables. For security reasons, use a different password for

>    DB_PASS (note DB_PASS for later use when connecting to database):

Reviewer: ```suggestion
   `$DB_PASS` and note it for future use:
```

        --cpu-count=8 \
        --region=$REGION \
        --cluster=$CLUSTER \
        --project=$PROJECT_ID \

>         --ssl-mode=ALLOW_UNENCRYPTED_AND_ENCRYPTED

Reviewer 2: Let's try to remember to remove this once we have proper cloud sql and alloydb connectors being used.

    AlloyDB. You may wish to open a new terminal to connect with.

1. Verify you can connect to your instance with the `psql` tool. Enter

>    password for AlloyDB (DB_PASS environment variable above)  when prompted:

Reviewer: ```suggestion
   password for AlloyDB (`$DB_PASS` environment variable set above) when prompted:
```
